src/components/Match/MatchStory.jsx

```diff
@@ -46,7 +46,7 @@ const TeamSpan = isRadiant => (
 
 // Modified version of PlayerThumb
 const PlayerSpan = (player) => {
-  if (!player || !heroes[player.hero_id]) {
+  if (!player || (!player && !heroes[player.hero_id])) {
```

Comment:
I don't think this change is needed

Reply:
I don't exactly understand what's happening but if I remove this change then the Story tab crashes.

example: matches/3564865624/story